### PR TITLE
Blog - Add veiw

### PR DESCRIPTION
**Pull request by Emmy-github-webdev**
## Blog - Add view

In this project, I implement view to display posts by a user, comments

### CODE REVIEW

File: app/controllers/blogs_controller.rb [hunk 1]
```diff
@@ -1,5 +1,12 @@
 class BlogsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @blogs = @user.recent_posts
```
Comment (Dipeshtwis) [inline app/controllers/blogs_controller.rb]: - [ ] Here in the blog's index, you have to show all the blogs written by the users and not only the recent blog with the 5 recent comments.

File: app/views/blogs/index.html.erb [hunk 2]
```diff
@@ -1 +1,21 @@
...
+   <div class="card-body">
+   <h1 class="card-title"> <%= blog.title %></h1>
+   <p class="card-content"><%= blog.text %></p>
+    <em class="card-post">
+    Comments: <%= blog.commentsCounter%>, Likes: <%= blog.likesCounter%>
+      <% blog.recent_comment%>
+    <em>
+   </div>
+  <% end %>
+</div>
```
Comment (Dipeshtwis) [inline app/views/blogs/index.html.erb:20]: - [ ] Also Please give the link here to go to the blog's show page. Currently, I won't able to go to blog show page.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Emmy-github-webdev, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N.A.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/blogs_controller.rb (hunk 1 above)
Comment (Emmy-github-webdev) [reply app/controllers/blogs_controller.rb]: Thank you. It is fix

File: app/views/blogs/index.html.erb (hunk 2 above)
Comment (Emmy-github-webdev) [reply app/views/blogs/index.html.erb:20]: Fix. Thanks

**Review by o-rashad — APPROVED** [review]
Asslam Alaikum @Emmy-github-webdev ,😊

Hi👋🏽, 
Your project is complete!  
There is nothing else to say other than... it's time to merge it 😎 

# Congratulations! 🎉 

Happy coding!👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Emmy-github-webdev** [comment]
Thank you

**Pull request merged**